ctapipe/io/simteleventsource.py

```diff
@@ -430,23 +432,25 @@ def _generate_events(self):
             yield data
 
     @staticmethod
-    def _fill_event_pointing(pointing, tracking_position):
+    def _fill_event_pointing(tracking_position):
         azimuth_raw = tracking_position["azimuth_raw"]
         altitude_raw = tracking_position["altitude_raw"]
         azimuth_cor = tracking_position.get("azimuth_cor", np.nan)
         altitude_cor = tracking_position.get("altitude_cor", np.nan)
 
         # take pointing corrected position if available
         if np.isnan(azimuth_cor):
-            pointing.azimuth = u.Quantity(azimuth_raw, u.rad)
+            azimuth = u.Quantity(azimuth_raw, u.rad, copy=False)
         else:
-            pointing.azimuth = u.Quantity(azimuth_cor, u.rad)
+            azimuth = u.Quantity(azimuth_cor, u.rad, copy=False)
 
         # take pointing corrected position if available
         if np.isnan(altitude_cor):
-            pointing.altitude = u.Quantity(altitude_raw, u.rad)
+            altitude = u.Quantity(altitude_raw, u.rad, copy=False)
         else:
-            pointing.altitude = u.Quantity(altitude_cor, u.rad)
+            altitude = u.Quantity(altitude_cor, u.rad, copy=False)
```

Comment:
and this one (basically the same thing to check)

Reply:
Can you provide a simtel file that has pointing corrections?